core/geometry/src/curve/internalContexts/CurveCurveIntersectXY.ts

```diff
@@ -1048,6 +1049,12 @@ export class CurveCurveIntersectXY extends RecurseToCurvesGeometryHandler {
         segmentA, this._extendA0, segmentA.point0Ref, 0.0, segmentA.point1Ref, 1.0, this._extendA1,
         this._geometryB, this._extendB0, this._extendB1, false,
       );
+    } else if (this._geometryB instanceof TransitionSpiral3d) {
+      const linestring = LineString3d.create();
+      this._geometryB.emitStrokes(linestring);
+      this.computeSegmentLineString(
+        segmentA, this._extendA0, this._extendA1, linestring, this._extendB0, this._extendB1, false,
```

Comment:
I don't think it makes sense to compute intersections to the _extended_ linestring approximant. However, in the next PR, when you refine these linestring-based intersection approximations with Newton, extending the spiral can be revisited (maybe the spiral API allows extensions up to a point?).

Reply:
Ok. This makes sense, I suppose, as the length of each spiral CurvePrimitive is baked into its definition.